src/components/fileviewer.js

```diff
@@ -104,10 +120,7 @@ export default class FileViewerContainer extends Component {
       });
     });
 
-    const fetchStatus = this.state.fetchStatus;
-    fetchStatus.testCoverage = true;
     this.setState({
```

Comment:
instead of communicating via `this`, make `parseCoverage()` a function, and do the setting in the caller

```javascript
return {coverage: {
         coveredLines: _.uniq(covered),
         uncoveredLines: _.uniq(uncovered),
         allTests: data,
         testsPerHitLine,
         testsPerMissLine,
}};
```